robottelo/constants.py

```diff
@@ -1187,3 +1187,31 @@
     u'pulp_data.tar',
     u'pulp.snar',
 ]
+
+TOOLS_ERRATA_DETAILS = [
+    ['Advisory', 'RHBA-2016:1503'],
+    ['CVEs', 'N/A'],
+    ['Type', 'Bug Fix Advisory'],
+    ['Severity', 'None'],
+    ['Issued', '7/27/16'],
+    ['Last Updated On', '7/27/16'],
+    ['Reboot Suggested', 'No'],
+    [
+        'Topic',
+        'Red Hat Satellite 6.2 now available for Red Hat Enterprise Linux 6 '
+        'and 7'
+    ],
+    ['Description', 'This update provides Satellite 6.2 client tooling'],
+    [
+        'Solution',
+        'Before applying this update, make sure all previously released errata'
+    ],
+]
+
+TOOLS_ERRATA_TABLE_DETAILS = [
+    'RHBA-2016:1503',
+    'Satellite 6.2 Tools Release',
+    'Bug Fix Advisory',
+    '0 Applicable, 0 Installable',
```

Comment:
a) and b) are not good enough approaches from test isolation perspective
About c), don't see any solution without changes to generic logic which I am not gonna do for sure, so probably will use 'Applicable' or 'Installable' string which will decrease coverage a bit, but keep stability on a higher level